liaisons/opcf/tech_reqs.md

```diff
@@ -0,0 +1,23 @@
+# Technical Requirements for OPC UA/WoT Collaboration
+
+## Use Cases
+See https://w3c.github.io/wot-usecases/#use-case-opc-ua-binding
+
+## Objectives
+1. Use OPC UA servers (device, node) from WoT Consumers; treat an OPC UA server as a Thing
+2. Access WoT TD metadata for OPC UA servers - integrate discovery process with OPC UA
+3. ? Let OPC UA servers connect to WoT Things; but if OPC UA devices are ALSO WoT Consumers, 1 satisfies this; --> "OPC UA Servients"
+
+## Deliverables
+1. OPC UA Ontology
+   - Defines information model and vocabulary for all OPC UA definitions and protocol options used in UA NodesetFile
+   - RDF/TTL/JSONLD Context file and URL, suitable for use in JSON-LD and RDF
+   - Note: useful for activites other than 2
+2. WoT TD Protocol Binding for OPC UA Nodesets
+   - Depends on 1 (TDs with this protocol binding would include the OPC UA ontology defined in 1)
+3. Process to transform a OPC UA NodesetFile to a WoT Thing Description (and vice versa, if TD describes OPC UA Nodeset)
```

Comment:
Automatic process or just a textual description?
Create a tool?

Reply:
the mapping should be specified. Tooling can then follow this approach.

Reply:
I think our goal should be a formal definition of the mapping so that it can be performed automatically.  As a standards activity we would not be creating tools ourselves as a deliverable, but members should probably do so as test cases.

We might have to do certain things manually though, e.g. generate TMs for OPC UA device classes.

Reply:
made a few changes to try and make this more explicit